universidade-de-sao-paulo-instituto-de-matematica-e-estatistica.csl

```diff
@@ -168,8 +168,8 @@
             <text variable="title"/>
             <text variable="number" prefix="U.S. Patent #"/>
             <date variable="issued">
-              <date-part name="month" form="short" suffix=". " strip-periods="true"/>
               <date-part name="day" suffix=", "/>
+              <date-part name="month" form="short" suffix=".-"/>
               <date-part name="year"/>
```

Comment:
@igorsteinmacher, I assume `<date-part name="day" suffix=", "/>` here on line 171 should become `<date-part name="day" suffix="-"/>` as well?

Reply:
You are right @rmzelle. Will update accordingly.
Thanks a lot for the review